### PR TITLE
fix: Add template file for php option class

### PR DESCRIPTION
**Pull request by rohith-prakash**
# Fixes #

Adds template files for php option class

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] Run `make test-docker`
- [ ] Verify affected language:
    - [ ] Generate [twilio-go](https://github.com/twilio/twilio-go) from our [OpenAPI specification](https://github.com/twilio/twilio-oai) using the [build_twilio_go.py](./examples/build_twilio_go.py) using `python examples/build_twilio_go.py path/to/twilio-oai/spec/yaml path/to/twilio-go` and inspect the diff
    - [ ] Run `make test` in `twilio-go`
    - [ ] Create a pull request in `twilio-go`
    - [ ] Provide a link below to the pull request
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai-generator/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

**Review by charan678 — COMMENTED** [review]
Don't change TwilioGeneratorTest file

**Review by charan678 — COMMENTED** [review]
You created lot of options files, which will be difficult to maintain. try to refactor this

File: src/main/resources/twilio-php/options/optionsCreateClass.mustache [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {{/isPathParam}}{{/optionalParams}}
+
+    /**
+    * Provide a friendly representation
+    *
+    * @return string Machine friendly representation
+    */
+    public function __toString(): string {
+        $options = \http_build_query(Values::of($this->options), '', ' ');
+        return '[Twilio.{{domainName}}.{{version}}{{#namespaceSubPart}}.{{namespaceSubPart}}{{/namespaceSubPart}}.Create{{apiName}}Options ' . $options . ']';
```
Comment (isha689) [inline src/main/resources/twilio-php/options/optionsCreateClass.mustache:27]: Is {{namespaceSubPart}} needed here and for other toString() methods ?

File: src/main/resources/twilio-php/options/optionsDeleteClass.mustache [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        {{#optionalParams}}{{^isPathParam}}
+        $this->options['{{paramName}}'] = ${{paramName}};
+        {{/isPathParam}}{{/optionalParams}}
+    }
+
+    {{#optionalParams}}{{^isPathParam}}
+{{>options/optionClassSetterComment}}
+    public function set{{#lambda.titlecase}}{{paramName}}{{/lambda.titlecase}}(string ${{paramName}}): self {
+        $this->options['{{paramName}}'] = ${{paramName}};
+    return $this;
```
Comment (isha689) [inline src/main/resources/twilio-php/options/optionsDeleteClass.mustache:16]: {{paramName}} needs to be camelized

**Pull request closed without merging**